Week3/homework/step3-bonus.js

```diff
@@ -3,8 +3,8 @@
 const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 
 function makeUnique(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  arr = [...new Set(arr)];
```

Comment:
👏 for using Sets. But avoid mutating/changing values of function arguments. Instead, create a local variable in the function as:

```
const uniqueArr = [...new Set(arr)];
return uniqueArr;
```

Reply:
okay 👍